assets/src/edit-story/output/page.js

```diff
@@ -55,10 +55,7 @@ function OutputPage({ page }) {
             <OutputElement key={'el-' + element.id} element={element} />
           ))}
         </div>
-        <div
-          className="page-safe-area"
-          style={{ ...aspectRatioStyles, ...backgroundStyles }}
-        >
+        <div className="page-safe-area" style={{ ...aspectRatioStyles }}>
```

Comment:
Intentional? 🤔

Reply:
Yep, it's already assigned above, it was previously assigned in two places which basically covered the background image.

Reply:
@swissspidy I actually think that perhaps the `backgroundStyles` should be set on the `<amp-story-page>`, not on any specific layer. WDYT? (unrelated to this PR, of course).

Reply:
That might make things more clear 👍 We can give it a shot